applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -77,6 +62,58 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 You cannot use this module with the :ref:`caf_ble_smp`.
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+You have to choose backend for the DFU process by enabling one of the following options:
+
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT` - This option depends on the MCUboot bootloader support for the DFU process.
+    For details, see :ref:`mcuboot` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_DESKTOP_DFU_BACKEND_SUIT` - This option depends on the SUIT support for the DFU process.
+    For details, see :ref:`ug_nrf54h20_suit_dfu` in the Zephyr documentation.
+
+.. note::
+    B0 bootloader is not integrated with MCUmgr.
+
+.. _nrf_desktop_dfu_mcumgr_mcuboot:
+
+MCUmgr configuration using MCUboot backend
+==========================================
+
+The MCUboot backend configuration selects the following options:
+
+* MCUmgr groups:
+
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG` - This option enables the MCUmgr image management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_MUTEX`
+  * :kconfig:option:`CONFIG_MCUMGR_GRP_OS` - This option enables the MCUmgr OS management handlers, which are required for the DFU process.
+    For details, see :ref:`zephyr:device_mgmt` in the Zephyr documentation.
+
+* :kconfig:option:`CONFIG_IMG_MANAGER`
+* :kconfig:option:`CONFIG_STREAM_FLASH`
+* :kconfig:option:`CONFIG_MCUBOOT_BOOTUTIL_LIB`
+
+To use the module with this backend, you must also enable the :kconfig:option:`SB_CONFIG_BOOTLOADER_MCUBOOT` Kconfig option.
```

Comment:
Double check if we should use `:kconfig:option:` syntax for Sysbuild options

Reply:
we are still waiting for tech writers to discuss it. Lets leave it for their review

Reply:
For now, we need to just have them between double backticks.